team.html

```diff
@@ -79,7 +78,7 @@ <h1 class="text-center" style="color:white; font-size: 60px">Meet Our Team</h1>
 
 
             <div class="box">
-                <div class="profile-img"><img class="img-fluid" src="images/team/2.jpg" style="border-radius: 50%;">
+                <div class="profile-img"><img class="img-fluid" src="images/team/2.jpg" alt="team member" style="border-radius: 50%;">
```

Comment:
Instead of keeping team member keep the member name in the alt tag.

Reply:
okay